mavros/08cd181/08cd181.bug

```diff
@@ -14,8 +14,8 @@ severity: error
 links: []
 bug:
   phase: build-time
-  specificity: "general issue: missing header"
-  architectural-location: N/A
+  specificity: "general issue"
```

Comment:
Minor nitpick: There's no need to escape this string or the one below it.

Reply:
minor, but easy to address (done, below)